ser/manage_exp_dir.py

```diff
@@ -0,0 +1,32 @@
+from dataclasses import asdict
+from datetime import date,datetime
+from pathlib import Path
+import json
+import torch
+
+PROJECT_ROOT = Path(__file__).parent.parent
+EXP_DIR = PROJECT_ROOT / "results"
+
+def create_exp_dir(name, exp_dir=EXP_DIR):
+
+    today = date.today().strftime("%d%m%Y")
```

Comment:
kinda dosn't matter but you can just do this is all one line. datetime has both date and time, so you can just do:

```datetime.now().strftime(%d-%m-%Y-%H-%M)```

Reply:
generall this is perfect though, great stuff.